mockgen/internal/tests/const_array_length/input.go

```diff
@@ -0,0 +1,8 @@
+//go:generate mockgen -package const_length -destination mock.go -source input.go
```

Comment:
nit: could you move this line below the package declaration. Having it here will have it show up in godoc. 

I know it is done this way in other packages, but I would like to refactor these sometime.